.github/workflows/releases-json.yml

```
@@ -0,0 +1,55 @@
+name: releases-json
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  release:
+    types:
+      - released
+  pull_request:
+    paths-ignore:
+      - '.github/releases.json'
+
+jobs:
+  generate:
+    uses: crazy-max/.github/.github/workflows/releases-json.yml@002654044825b3c2b9856af61b8a2aaf389706b1
+    with:
+      repository: docker/buildx
+      artifact_name: releases-json
+      filename: releases.json
+    secrets: inherit
+
+  open-pr:
+    runs-on: ubuntu-22.04
+    if: github.event_name == 'release'
+    needs:
+      - generate
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Download
+        uses: actions/download-artifact@v3
+        with:
+          name: releases-json
+          path: .github
+      -
+        name: Commit changes
+        run: |
+          git add -A .
+      -
+        name: Create PR
+        uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04
+        with:
+          base: master
+          branch: releases-json/${{ github.event.release.name }}
+          commit-message: "update .github/releases.json"
```

```suggestion
          commit-message: "github: update .github/releases.json"
```

Or some other prefix, for consistency.

---

`github` prefix sounds good